src/assets/styles/antd/antd-typography.scss

```diff
@@ -0,0 +1,4 @@
+h2.ant-typography,
+div.ant-typography {
+  margin-bottom: 0;
+}
```

Comment:
Probably we need to add stylelint. We have already done it in out vue project. And we need to resolve issues with empty lines in scss files. It should be resolved automatically in the end.